nestkernel/random_manager.cpp

```diff
@@ -171,8 +171,8 @@ nest::RandomManager::set_status( const DictionaryDatum& d )
 
   if ( rng_seed_updated or rng_type_updated )
   {
-    finalize( /* reset_kernel */ false );
-    initialize( /* reset_kernel */ false );
+    finalize( /* adjust_number_of_threads_only */ true );
```

Comment:
```suggestion
    finalize( /* adjust_number_of_threads_or_rng_seed_only */ true );
```
debates about named parameters to function calls aside (maybe in a future C++ standard?), should this parameter name be consistent along managers, or did you name it in more specific ways for specific managers?